gulp/gulpfile.js

```diff
@@ -0,0 +1,67 @@
+const gulp = require('gulp');
+const sass = require('gulp-sass')(require('sass'));
+const browserSync = require('browser-sync').create();
+const concat = require('gulp-concat');
+const autoprefixer = require('gulp-autoprefixer');
+const htmlmin = require('gulp-htmlmin');
+const minify = require('gulp-minify');
+const cleanCSS = require('gulp-clean-css');
+
+function liveReload() {
+  browserSync.init({
+    server: './src'
+  });
+
+  gulp.watch('src/sass/*.scss', cssTranspile);
+  gulp.watch('src/scripts/*.js', jsCompile);
+  gulp.watch("src/*.html").on('change', browserSync.reload);
+};
+
+function cssTranspile() {
+  return gulp.src('./src/sass/main.scss')
+    .pipe(sass().on('error', sass.logError))
+    .pipe(gulp.dest('./src/css'));
+};
+
+function jsCompile() {
+  return gulp.src('./src/scripts/*.js')
+    .pipe(concat('script.js'))
+    .pipe(gulp.dest('./src/js'));
+};
+
+function cssPostprefixer(done) {
+  return gulp.src('./src/css/*.css')
+    .pipe(autoprefixer({
+      cascade: false
+    }))
+  .pipe(gulp.dest('css'));
```

Comment:
не правильно указан путь, в таком варианте создается папка css в корне проекта, а должна быть папка с файлом внутри dist
 ```suggestion
  .pipe(gulp.dest('dist/css'));
```